supervisor/mock/config.example.toml

```diff
@@ -1,6 +1,6 @@
 [base]
 supervisor_id = "7d55ec6d-15e7-4b84-8c04-7c085fe60df4"
-coord_connector = "cli_connector"
+coord_connector = "ws_connector"
```

Comment:
I think leaving this set to the CLI connector by default is a good idea -- it makes it easier to test local changes. We should be able to override this parameter in the CI workflow?